website/server/controllers/api-v3/user.js

```diff
@@ -520,6 +520,101 @@ api.getUserAnonymized = {
 
 const partyMembersFields = 'profile.name stats achievements items.special';
 
+async function castTaskSpell (res, req, targetId, user, spell) {
```

Comment:
can we move those to `ops/spells` ?

Reply:
ops in /common? that should only include stuff used both on client and server

Reply:
hmm I see, well sooner or later we'd need to clean up / split `user.js` since its getting tooo long ^^ , what about `api-v3/user/spell.ops.js`  or something

Reply:
My plan is to move stuff to libraries and create modules we can share. Common is bloated and unorganized so I'd like to keep stuff out of there.

I'll be working splitting the code. I'm on Challenges right now.